FpCategories/PackageInfo.g

```diff
@@ -30,6 +30,20 @@ Persons := [
     Place := "Siegen",
     Institution := "University of Siegen",
   ),
+  rec(
```

Comment:
Oh, I am so sorry I forgot that. I copied one of the existing packages in CategoricalTowers and used it as a template.